app/models/ExportsDeclaration.scala

```diff
@@ -47,6 +47,12 @@ case class ExportsDeclaration(
   seals: Seq[Seal] = Seq.empty
 ) {
   def itemBy(itemId: String): Option[ExportItem] = items.find(_.id.equalsIgnoreCase(itemId))
+
+  def updatedItem(itemId: String, update: ExportItem => ExportItem): ExportsDeclaration =
```

Comment:
I'd name it `updateItem`. Also why does it pass an `ExportItem => ExportItem` rather than just an `ExportItem`?

Reply:
We have immutable structure - in `scala.collection.Map` there is method `updated` and here i am tend to mimic that convention.

Having just setter here would break hermetization - passing function for updating existing object does not reveal any information about internal structure of Declaration and ExporItems and could be used as setter simply by ignoring argument.